jep/236/README.adoc

```diff
@@ -229,15 +229,14 @@ Review link:https://issues.jenkins.io/issues/?jql=resolution%20%3D%20Unresolved%
 
 We will track those issues in the link:https://issues.jenkins.io/browse/JENKINS-67688[JENKINS-67688 epic].
 
-==== Timeline (TBD)
-
-* Announce Java 8 end of support for weekly in blogpost
-** Describe Java 11 upgrade process for users of war, deb, rpm, and msi installations
-* Add higher visibility warning on Java 8 end of support in Jenkins weekly (an idea that is not yet fully formed)
-* Remove Java 8 from weekly core release, weekly Docker controller images
-* Announce Java 8 end of support for LTS in blogpost
-* Remove Java 8 from LTS core release, include in changelog and upgrade guide
-* Remove Java 8 from Docker agent images
```

Comment:
I don't see a need to mention these here, they should just track weekly and LTS removals when maintainers get to them

Reply:
Agreed